gravitee-apim-console-webui/src/entities/plugin/PluginMoreInformation.ts

```diff
@@ -13,8 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-export type PluginMoreInformation = {
-  description?: string;
-  schemaImg?: string;
-  documentationUrl?: string;
-};
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Information } from '../../management/api/component/gio-information-dialog/gio-information-dialog.component';
+
+export type PluginMoreInformation = Information;
```

Comment:
shouldn't it be the opposite ? Declare here the type and use it in the component ? 🤔